### ISSUE TITLE
JS error Transfer error ETH from one account to another from Parity UI

### ISSUE THREAD

**Issue by vincentserpoul** [ethcore/parity#3849]
I'm using Parity/v1.5.0-unstable-03db4d037-20161215/x86_64-macos/rustc1.13.0
chrome 54.0.2840.98 (64-bit)

```
Uncaught Error: new BigNumber() not a number: undefined
    at D (http://127.0.0.1:8182/vendor.js:31:5244)
    at http://127.0.0.1:8182/vendor.js:31:10949
    at new t (http://127.0.0.1:8182/vendor.js:31:3151)
    at eval (webpack:///./api/format/input.js?:155:37)
    at Array.forEach (native)
    at inOptions (webpack:///./api/format/input.js?:146:88)
    at Parity.postTransaction (webpack:///./api/rpc/parity/parity.js?:248:128)
    at TransferStore.send (webpack:///./modals/Transfer/store.js?:439:40)
    at Object.eval (webpack:///./modals/Transfer/store.js?:560:6051)
    at N (http://127.0.0.1:8182/vendor.js:16:29328)
```

**Comment by jacogr** [comment]
Just to help us track this -

1. Did you change/specify any gas via the "advanced options"
2. Did you send any ETH along with the transaction? (value in the ETH box)
3. Anything else you can think of that is out of the ordinary?

**Comment by vincentserpoul** [comment]
here is my conf json:

```
{
    "name": "Development",
    "engine": {
        "InstantSeal": null
    },
    "params": {
        "accountStartNonce": "0x0100000",
        "maximumExtraDataSize": "0x20",
        "minGasLimit": "0x1388",
        "networkID": "0x2"
    },
    "genesis": {
        "seal": {
            "generic": {
                "fields": 0,
                "rlp": "0x0"
            }
        },
        "difficulty": "0x20000",
        "author": "0x0000000000000000000000000000000000000000",
        "timestamp": "0x00",
        "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
        "extraData": "0x",
        "gasLimit": "0x2fefd8"
    },
    "accounts": {
        "0000000000000000000000000000000000000001": {
            "balance": "1",
            "nonce": "1048576",
            "builtin": {
                "name": "ecrecover",
                "pricing": {
                    "linear": {
                        "base": 3000,
                        "word": 0
                    }
                }
            }
        },
        "0000000000000000000000000000000000000002": {
            "balance": "1",
            "nonce": "1048576",
            "builtin": {
                "name": "sha256",
                "pricing": {
                    "linear": {
                        "base": 60,
                        "word": 12
                    }
                }
            }
        },
        "0000000000000000000000000000000000000003": {
            "balance": "1",
            "nonce": "1048576",
            "builtin": {
                "name": "ripemd160",
                "pricing": {
                    "linear": {
                        "base": 600,
                        "word": 120
                    }
                }
            }
        },
        "0000000000000000000000000000000000000004": {
            "balance": "1",
            "nonce": "1048576",
            "builtin": {
                "name": "identity",
                "pricing": {
                    "linear": {
                        "base": 15,
                        "word": 3
                    }
                }
            }
        },
        "cf70c68d669c795d8cf11e1e85e209cf3432432432": {
            "balance": "3000000000000000000000",
            "nonce": "1048576"
        }
    }
}
```

**Comment by ngotchac** [comment]
Might be fixed with #3990

**Comment by vincentserpoul** [comment]
I updated to the latest master and still have the issue

a._onMessage @ :8182/index.9ff9f7e44e.js:10
:8182/#/accounts/0xcf70C68D669c795D8CF11e1E85E209Cf71927541:1 Uncaught (in promise) t
:8182/vendor.js:14 Uncaught Error: new BigNumber() not a number: undefined
    at P (http://127.0.0.1:8182/vendor.js:31:17139)
    at http://127.0.0.1:8182/vendor.js:31:22844
    at new t (http://127.0.0.1:8182/vendor.js:31:15046)
    at http://127.0.0.1:8182/index.9ff9f7e44e.js:3:17128
    at Array.forEach (native)
    at f (http://127.0.0.1:8182/index.9ff9f7e44e.js:3:17027)
    at e.value (http://127.0.0.1:8182/index.9ff9f7e44e.js:10:13167)
    at e.value (http://127.0.0.1:8182/index.9ff9f7e44e.js:16:12161)
    at Object.<anonymous> (http://127.0.0.1:8182/index.9ff9f7e44e.js:16:17429)
    at j (http://127.0.0.1:8182/vendor.js:16:29060)
